### PR TITLE
build: Add a new gradle task to start the docker container in debug mode.

### PR DESCRIPTION
**Pull request by AlfredoG87**
**Description**:
Add a new gradle task to start the docker container in debug mode.

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by mattp-swirldslabs — COMMENTED** [review]
I think we need a gradle task to trigger the debug flow

File: server/docker/update-env.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # This script is called by gradle and get the current project version as an input param
 
 if [ $# -lt 1 ]; then
-  echo "USAGE: $0 <VERSION>"
+  echo "USAGE: $0 <VERSION> <DEBUG>"
```
[the remark this answers is not in the document]
Comment (AlfredoG87) [reply server/docker/update-env.sh:7]: yes, I just updated the PR

[the remark this answers is not in the document]
Comment (mattp-swirldslabs) [reply server/docker/update-env.sh:7]: Looks good!

**Review by mattp-swirldslabs — APPROVED** [review]
LGTM

**Pull request merged**